src/Components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const Calculator = () => (
```

Comment:
- [ ] Your _Calculator_ component _(Calculator.js)_ is not classed based. Kindly change this component to a class-based component.